mixins/models.py

```diff
@@ -134,6 +134,7 @@ class URLMixin(models.Model):
         null=True,
         on_delete=models.SET_NULL,
     )
+    new_tab = models.BooleanField(default=False, help_text="Open this link in a new tab/window.")
```

Comment:
Make migrations?

Reply:
Yeah, I thought that too. Had a quick chat with Will and `giant-mixins` doesn't implement the models, they're abstract.

Reply:
Ah yeah if there are no concrete models in this package using them then it won't have any migration changes, that's all good then